tests/ui/places/test_places_api.py

```
@@ -245,3 +245,33 @@ def test_create_place_record(app, place_idref_data, place_gnd_data, tmpdir):
         "pid": "1",
         "type": "bf:Place",
     }
+
+
+def test_places_utils_get_places_endpoints(app):
```

Ruff (0.15.2)

[warning] Unused function argument: `app`

(ARG001)